tests/app_builder/with_bank.rs

```diff
@@ -0,0 +1,66 @@
+use anyhow::Result as AnyResult;
+use cosmwasm_std::{
+    Addr, Api, BankMsg, BankQuery, Binary, BlockInfo, CustomQuery, Querier, Storage,
+};
+use cw_multi_test::{AppBuilder, AppResponse, Bank, BankSudo, CosmosRouter, Module};
+use schemars::JsonSchema;
+use serde::de::DeserializeOwned;
+use std::fmt::Debug;
+
+struct MyBankKeeper {}
+
+impl Module for MyBankKeeper {
```

Comment:
All of the tests share the same implementation, so I would just use the same keeper in all of them.
That will shorten the code significantly (and maybe it makes more sense at that point to move the tests to the same file as the keeper)

Reply:
I have removed all bloated code that in fact would duplicate tests we already have in every single module or keeper. I just simplified the test cases, just to make sure that the `App` is build and initialized using `AppBuilder` (that was the goal of introducing these tests).